.github/workflows/rcmd.yml

```diff
@@ -27,12 +27,10 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          # for latest stable version of R for MacOSX
-          - {os: macOS-latest, r: 'release'}
+          # test on one version of R for MacOSX
           - {os: macOS-latest, r: '3.6'}
```

Comment:
I feel like if we're going to only test on one version, it should be the 'release', not '3.6'?

Reply:
👌